### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by krate27**
Fetched and video listed on web page. Made layout responsive.

Screenshot (666)
Screenshot (667)

### CODE REVIEW

**Review by hinanshisuthar — COMMENTED** [review]
Everything looks good to me. Clean code written. Good to merge.

File: src/Components/VideoCard/video-card.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import '../VideoCard/video-card.css';
+const VideoCard = ({ video }) => {
+    const {_id,title,catchName,uploaded} = video
+  return (
+    <>
+      <div class="container">
+        <figure class="menu-card">
+          <img src={`https://i.ytimg.com/vi/${_id}/0.jpg`}/>
+          <figcaption>
```
Comment (ashishnnnnn) [inline src/Components/VideoCard/video-card.js:9]: Images can have alt attribute..

File: src/Components/VideoCard/video-card.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import '../VideoCard/video-card.css';
+const VideoCard = ({ video }) => {
+    const {_id,title,catchName,uploaded} = video
+  return (
+    <>
+      <div class="container">
+        <figure class="menu-card">
```
Comment (ashishnnnnn) [inline src/Components/VideoCard/video-card.js:7]: Class should be changed to className..

File: src/custom-hooks/useFetch.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
```
Comment (ashishnnnnn) [inline src/custom-hooks/useFetch.js:3]: Liked the implementation of the context Hooks..

**Review by ashishnnnnn — APPROVED** [review]
Good Work..
Nice usages of Hooks..

**Review by lomesshh — APPROVED** [review]
clean and modular code
UI looks good
looks good to merge

**Pull request merged**